core/storage/statistics/gae_models.py

```diff
@@ -696,6 +893,21 @@ def _shard_answers(
         list, shard the answers such that a list of answer lists are returned.
         The first entry is guaranteed to contain all answers of the current
         answer list.
+
+        Args:
+            current_answer_list: list(Json). list of answer dicts each of which
```

Comment:
Json isn't a valid python type; what do you get when you do `type(x)` when x is an element in the list?

Reply:
changed to list(dict).